### DISCUSSION TITLE
Vesting NFT fractionalizer with Shell Protocol

### DISCUSSION THREAD

**Discussion by viraj124** [sablier-labs/evm-monorepo#782] [Show and tell]
We have built a fractionalizer as a usecase for the vesting nft's where instead of having 1 nft that is created upon vesting users can  vest small amounts and thus have multiple vesting nft and can use the fractionalizer to get fungible tokens based on the value of the underlying token their vesting nft holds.

users interact through our [ocean protocol](https://wiki.shellprotocol.io/how-shell-works/the-ocean-accounting-hub) which is a common accounting layer for all our primitives and swap the nft for fungible tokens and vice versa

**Comment by PaulRBerg** [comment]
Hey @viraj124, thanks for sharing your integration.

This is very cool!
